File: 08-09-2023/calculator.py
while 1:
    print ("Enter number 1:")
    a = input()
    print ("Enter number 2:")
    b = input()
    print ("What to do:")
    c = input()
    if b == '0' and c == '/':
            print("You cant do this.")
            exit();
    a = int(a)
    b = int(b)
    if c == '+':
            print (a+b)
    if c == '-':
            print (a-b)
    if c == '*':
            print (a*b)
    if c == '/':
            print (a/b)
    print ("calculate something else? [Y/n]")
    cont = input()
    if cont != 'y':
        exit()

File: 08-09-2023/calculator.cpp
#include <iostream>

using namespace std;

int main()
{
    double a, b;
    char c;
    while (1) {
        cout << "Enter number 1:";
        cin >> a;
        cout << "Enter number 2:";
        cin >> b;
        cout << "What to do?";
        cin >> c;
        if (c == '+') cout << a+b << endl; else
        if (c == '-') cout << a-b << endl; else
        if (c == '*') cout << a*b << endl; else
        if ((c == '/') && (b != 0)) cout << a/b << endl; else
        cout << "Error\n";
        cout << "Calculate something else? [Y/n]\n";
        cin >> c;
        if (c != 'y') {
            cout << "OK\n";
            exit(0);
        }
    }
}
